src/transformers/models/llama/modeling_llama.py

```diff
@@ -1100,6 +1109,7 @@ def forward(
         input_ids: torch.LongTensor = None,
         attention_mask: Optional[torch.Tensor] = None,
         position_ids: Optional[torch.LongTensor] = None,
+        cache_position: Optional[torch.LongTensor] = None,
```

Comment:
I think we should put the `cache_position` positonal argument at the end. This might be breaking for users that call our public modules with no explicit positional arguments

Reply:
go point, fixing this